README.md

```diff
@@ -67,8 +67,10 @@ A index.html is provided in the web folder to preview the layers in mapproxy.
 
 # Reverse proxy
 
-The mapproxy container 'speaks' ``uwsgi`` so you need to put nginx in front of it
-(try the ``nginx docker container``). A sample configuration (via linked
+The mapproxy container can 'speaks' ``uwsgi`` protocol so you can also put nginx in front of it 
+to receive http request and translate it to uwsgi
+(try the ``nginx docker container``). However our sample configuration by default 
+make `uwsgi` uses `http` socket instead of `socket` parameter (uwsgi protocol). A sample configuration (via linked
```

Comment:
I just thought this is outdated because the current `uwsgi.ini` uses http socket instead of the usual `uwsgi` socket.